tests/Provider/Psr17ProviderTest.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Yii\Web\Tests\Provider;
+
+use Psr\Http\Message\RequestFactoryInterface;
+use Psr\Http\Message\ResponseFactoryInterface;
+use Psr\Http\Message\ServerRequestFactoryInterface;
+use Psr\Http\Message\StreamFactoryInterface;
+use Psr\Http\Message\UploadedFileFactoryInterface;
+use Psr\Http\Message\UriFactoryInterface;
+use Yiisoft\Yii\Web\Tests\TestCase;
+
+final class Psr17ProviderTest extends TestCase
+{
+    public function testProviderConfig(): void
+    {
+        $this->assertNotEmpty($this->container->get(RequestFactoryInterface::class));
```

Comment:
It's better to use `assertInstanceOf` for these.

Reply:
done.